src/core/core.hh

```diff
@@ -506,7 +506,8 @@ namespace SSC {
             const String seq,
             const String path,
             int mode,
-            Module::Callback cb
+            Module::Callback cb,
+            bool recursive = false
```

Comment:
Could we put those before `cb` and after `mode`?

Reply:
I tried that but clang got mad about having defaulted arguments come before the callback.

Reply:
lets just avoid the default value here and pass the recursive option where ever this function is used